contracts/smart-contract-wallet/SmartAccount.sol

```diff
@@ -502,18 +517,18 @@ contract SmartAccount is
     //Method is updated to instruct delegate call and emit regular events
     function execFromEntryPoint(address dest, uint value, bytes calldata func, Enum.Operation operation, uint256 gasLimit) external onlyEntryPoint returns (bool success) {        
         success = execute(dest, value, func, operation, gasLimit);
-        require(success, "Userop Failed");
+        if(!success) revert ExecutionFailed();
     }
 
     function _requireFromEntryPointOrOwner() internal view {
-        require(msg.sender == address(entryPoint()) || msg.sender == owner, "account: not Owner or EntryPoint");
+        if(msg.sender != address(entryPoint()) && msg.sender != owner) revert CallerIsNotEntryPointOrOwner(msg.sender);
```

Comment:
maybe we should keep this require to align with simple account and entry point test cases.